src/vue/components/popup.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div :data-notification-visible="$store.state.ui.notification.visible && $store.state.ui.notification.visible"
+  <div :data-notification-visible="$store.state.ui.notification.visible"
+       :data-notification-auth-visible="$store.state.auth && $store.state.auth.notification.visible"
```

Comment:
ok, yes, good idea. Will do this

Reply:
oh, but actually, I'm not sure about this.  I was thinking that the inclusion of the Auth module would be optional, because we don't know yet how 3rd parties might or might not want to include authentication (and similarly, whether we ant to allow 3rd party access to our auth module). Need to think about this a bit.

Reply:
I added an isEnabled() api method to the auth data module, which returns false if has been initialized with a null auth object.